identity/app/views/repermissionJourney.scala.html

```diff
@@ -15,39 +18,85 @@
     availableLists: EmailNewsletters
 )(implicit request: RequestHeader, messages: play.api.i18n.Messages, context: model.ApplicationContext)
 
-@tab(i: Int, name: String, url: String, dataTestId: Option[String], hidden: Boolean = false, optionalClass: String = "") = {
-    <li class="tabs__tab @if(hidden){is-hidden} @if(page.metadata.id == url){tabs__tab--selected tone-colour tone-accent-border} @optionalClass" role="tab" id="tabs-account-profile-@i-tab" aria-selected="@(page.metadata.id == url)" aria-controls="tabs-account-profile-@i">
-        <a href="@url" data-tabs-href="#tabs-account-profile-@i" data-link-name="@url edit profile tab" data-pushstate-url="@url"
-            @dataTestId.map{idValue => data-test-id="@idValue"}>@name</a>
-    </li>
-}
-
-@content(i: Int, url: String, optionalClass: String = "")(body: => Html) = {
-    <div id="tabs-account-profile-@i"
-         class="@RenderClasses(Map(
-            "u-h" -> (page.metadata.id != url)
-         ), "tabs__pane u-cf @optionalClass")"
-         role="tabpanel"
-         aria-labelledby="tabs-account-profile-@i-tab"
-         data-link-name="Public Profile"
-         data-link-context="Identity/profile">
-        @body
-    </div>
-}
-
-@mainLegacy(page, projectName = Option("identity")){
-    <link rel="stylesheet" id="stripe-sprite" type="text/css" media="all" href="@Static("stylesheets/membership-icons.css")"/>
-}{
+@mainLegacy(page, projectName = Option("identity")) { } {
 
     <div class="identity-wrapper-for-bg">
         <div class="gs-container">
-            <div class="monocol-wrapper js-account-profile-forms">
+            <div class="monocol-wrapper">
+
+                <div class="u-cf identity-section identity-wrapper">
+
+                    <div class="js-errorHolder manage-account__errors"></div>
+
+                    <div class="manage-account-wizard manage-account-wizard--repermission" >
+                        <div class="manage-account-wizard__step">
+                        <center>
+                            <img width="100%" src="https://i.imgur.com/TqI7Ub8.png" />
```

Comment:
`alt=""` is missing. Also: we shouldn't link an external image service shouldn't we?